src/Components/Categories/Categories.js

```diff
@@ -1,13 +1,24 @@
-import React from 'react';
+import { useDispatch } from 'react-redux';
+import categoryActionReducer from '../../redux/categories/categories';
 import './Categories.css';
 
-const Categories = () => (
-  <div className="container catContainer">
-    <div className="row">
-      <hr className="col-12" />
-      <button type="button" className="col-4  my-2 py-5 px-3 catButton ">Check status</button>
+const Categories = () => {
+  const dispatch = useDispatch();
+  return (
+    <div className="container catContainer">
+      <div className="row">
+        <hr className="col-12" />
+        <button
+          type="button"
+          className="col-4  my-2 py-5 px-3 catButton "
+          onClick={() => {
+            dispatch(categoryActionReducer());
+          }}
+        >
+          Check status
+        </button>
```

Comment:
- [ ] Once more, based on the requirements of this project, this button should **use** a function `checkCategoriesStatus` (name can be different) made in the component. So you'll need to implement a function up :point_up: inside this component and use it inside this button